xss/xss.go

```diff
@@ -46,19 +44,15 @@ func bodyHandler(w http.ResponseWriter, r *http.Request, safety string) (templat
 		return template.HTML("Cannot GET " + r.URL.Path), false
 	}
 
-	err := r.ParseForm()
-	if err != nil {
-		return template.HTML(err.Error()), false
-	}
-	input := r.Form.Get("input")
+	inputs := utils.FormValueInput(r, utils.INPUT)
 
 	if safety == "safe" {
-		input = url.QueryEscape(input)
+		inputs = url.QueryEscape(inputs)
```

Comment:
`inputs` should be `input` since there is only a single string returned, not a slice of string